core/test/computed/metrics/total-blocking-time-test.js

```diff
@@ -30,9 +30,9 @@ describe('Metrics: TotalBlockingTime', () => {
       pessimistic: Math.round(result.pessimisticEstimate.timeInMs),
     }).toMatchInlineSnapshot(`
       Object {
-        "optimistic": 777,
-        "pessimistic": 777,
-        "timing": 777,
+        "optimistic": 2538,
+        "pessimistic": 2820,
+        "timing": 2679,
```

Comment:
Would it be better to use a more controlled trace to replace this? I don't envy the next person making changes to the simulator having to manually determine if any changes to these numbers are reasonable from a massive cnn trace.

Alternatively, if it needs to be more complex than what's in our smoke tests, in #14525 we had discussed using the test sites in https://github.com/ChromeDevTools/performance-stories as the source of generating test artifacts. Controlled, intentional, but also a decent variety of scenarios.